book/website/collaboration/new-community/new-community-guide.md

```diff
@@ -0,0 +1,90 @@
+- Make sure that the choices of these platforms are made to ensure that there is a low barrier to join them.
+
+### 2. Provide a project summary file:
+
+- The first document in your project should be a project summary file, which in a GitHub repository will be a [README.md file](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-readmes).
+- This will provide basic information about your project so that people can evaluate why your project will be interesting for them.
+  - Here is [a template](https://github.com/PurpleBooth/a-good-readme-template) by the GitHub user [PurpleBooth](https://github.com/PurpleBooth).
+- In this file, include what your project vision and goals are, why the project is useful, what are the possible milestones in the project, how a contributor or user can get started, who can they reach out for help, and what is currently missing in the project in terms of stakeholders, skills, or scope.
+- You can use emojis, GIFs, videos, or your personal narrative to make your project relatable.
+  - See [The Atom project](https://github.com/atom/atom) for example.
+
+### 3. Select a Code of conduct:
+
+- Add an Open Source Project [Codes of Conduct](https://opensourceconduct.com/) to your project.
+- This document should not be used as a token, it is very important to put intentional effort into it.
+- When using GitHub, you can add a “CODE_OF_CONDUCT.md” file on your GitHub repository.
+- List the expected and unacceptable behaviors, describe the reporting and enforcement process, explicitly define the scope, and use an inclusive tone  (see [GitHub instructions here](https://help.github.com/en/github/building-a-strong-community/adding-a-code-of-conduct-to-your-project)).
+- Whenever you update your code of conduct, invite comments from your members to ensure that their concerns are addressed.
+  - This can be done on [GitHub issues](https://help.github.com/en/github/managing-your-work-on-github/about-issues), or [Pull Requests](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests).
+
+### 4. Provide contribution guidelines and interaction pathways:
+
+- A thoughtful guideline helps people decide which pathway they can choose to contribute to your project, or if at all they want to be in your community.
+- Make sure that your community interactions and different pathways to contribute are open, inclusive, and clearly stated.
+  - If people can’t figure out how to contribute they will drop off without help.
+- Value different types of contributions - coding projects are not only about code, therefore list documentation and other management skills as well.
+- You can use {ref}`Persona Creation Tool<pd-persona-creation>` or [Persona and Pathway exercise](https://mozillascience.github.io/working-open-workshop/personas_pathways/) to brainstorm who could be your possible community members.
```

Comment:
Add a link to the new _Turing Way_ chapter on personas? 🙃

Reply:
This should do the trick: {ref}`Persona Creation Tool<pd-persona-creation>`